src/ray/raylet/node_manager.h

```diff
@@ -121,12 +121,12 @@ class NodeManager : public rpc::NodeManagerServiceHandler {
   ///
   /// \param data Data associated with the new client.
   /// \return Void.
-  void ClientAdded(const ClientTableData &data);
+  void ClientAdded(const GcsNodeInfo &data);
```

Comment:
It seems reasonable to rename this method to `NodeAdded` in this PR.

Reply:
I will rename class `ClientTableData` in a seperate pr. And I will rename methods at the same time. It that ok?